platform-core-module/src/main/java/com/softicar/platform/core/module/program/execution/ProgramExecutionRunnable.java

```diff
@@ -70,17 +82,20 @@ private AGProgramExecution updateStartedAt() {
 
 	private void executeProgram() {
 
-		try (LogOutputScope scope = new LogOutputScope(logBuffer)) {
+		try (var scope = new LogOutputScope(logOutput)) {
+			Log.finfo("Program started.");
 			new ProgramStarter(getProgramUuid()).start();
+			Log.finfo("Program finished.");
 		} catch (Throwable throwable) {
 			this.failed = true;
+			// TODO this seems to duplicate some log lines we already have
```

Comment:
do we? also note that the log scope is closed at this location

Reply:
Created internal issue `PLAT-1317` for this.
The observed behavior is not a regression due to this PR.